packages/xsnap/test/test-xsnap.js

```diff
@@ -48,18 +50,19 @@ test('evaluate until idle', async t => {
 test('evaluate infinite loop', async t => {
   const opts = options();
   const vat = xsnap(opts);
+  t.teardown(vat.terminate);
   await t.throwsAsync(vat.evaluate(`for (;;) {}`), {
-    message: 'xsnap test worker exited',
+    message: /exited with code 7/,
```

Comment:
This no longer validates the `xsnap` `name` option.

Might it make sense for the `xsnap` utility to convert machine error codes into humane error messages? There’s Node.js precedent for having an `error.code` property for automated recovery logic.

Reply:
> This no longer validates the `xsnap` `name` option.

The test label said it was about infinite loops, not the name option. But OK, this is a regression, so I un-did it in b34904522 .

> Might it make sense for the `xsnap` utility to convert machine error codes into humane error messages? There’s Node.js precedent for having an `error.code` property for automated recovery logic.

Perhaps... but I prefer to postpone it indefinitely as a separate issue: #2749 .